contexts/security-v3-unstable.jsonld

```diff
@@ -2,12 +2,53 @@
   "@context": [{
     "id": "@id",
     "type": "@type",
-    "@protected": true,
```

Comment:
Why are we dropping the protected value here?

Reply:
IIRC, tests confirmed that the definitions were not usable with it set to true. I suspect this is caused by the inability of some libraries to override the default security context, but you certainly can't mix the contexts with this set.

if we set this to true, the file will be unusable unless you can override `sec/v2 -> sec/v3-unstable` in your document loader.

Reply:
fixed